### PR TITLE
Document `--ethstats-cacert-file`

### LINKED ISSUE #1297: [Content] Document `--ethstats-cacert-file` and update `--ethstats` default
### Describe the issue

Document `--ethstats-cacert-file` CLI option and update `--ethstats` default to 443 for wss and 80 for ws. See https://github.com/hyperledger/besu/pull/5301. 

### What page has the issue?

https://besu.hyperledger.org/en/stable/public-networks/reference/cli/options/#ethstats

### Issue type

Missing content

### Change suggestion

_No response_

### More context

_No response_

### Besu documentation issue guidelines

- [X] I read and followed this project's documentation guidelines.

### PR DESCRIPTION
**Pull request by alexandratran**
## Pull request checklist

Use the following list to make sure your PR fits the Besu documentation quality standard.

### Before creating the pull request

Make sure that:

- [x] [All commits in this PR are signed off for the DCO](https://wiki.hyperledger.org/display/BESU/DCO).
- [x] You've read the [contribution guidelines](https://wiki.hyperledger.org/display/BESU/Documentation).
- [x] You've [previewed your changes locally](https://wiki.hyperledger.org/display/BESU/Preview+the+documentation).

## Describe the change

Document `--ethstats-cacert-file`. Document default ports for `--ethstats`. Remove note on `--ethstats-contact` because the usage is explained in the option description.

## Issue fixed

fixes #1297 

## Impacted parts

For content changes:

- [x] Documentation content
- [ ] Documentation page organization

For tool changes:

- [ ] Github Actions workflow
- [ ] Build and QA tools configuration (for example, lint rules or Vale style)
- [ ] MkDocs templates
- [ ] MkDocs configuration
- [ ] Python dependencies
- [ ] Node dependencies and JavaScript
- [ ] Read the Docs configuration
- [ ] GitHub integration

## After creating your PR and tests have finished

Make sure that:

- [ ] You've fixed any issues raised by the tests.
- [ ] You've [previewed your changes on Read the Docs](https://wiki.hyperledger.org/display/BESU/Preview+the+documentation)
  and added a [preview link](#preview).

### CODE REVIEW

**Review by usmansaleem — APPROVED** [review]
LGTM

File: docs/public-networks/reference/cli/options.md [hunk 1]
```diff
@@ -665,6 +665,37 @@ The default is `8551`.
...
+
+=== "Configuration file"
+
+    ```bash
+    ethstats-cacert-file="./root.cert"
+    ```
+
+Path to the root certificate authority (CA) certificate file of the Ethstats server specified by
+[`--ethstats`](#ethstats).
+This option is useful in non-production environments.
```
Comment (bgravenorst) [inline docs/public-networks/reference/cli/options.md:698]: Not sure I quite understand why it's useful in non-prod environments though.

**Review by bgravenorst — APPROVED** [review]
Looks good. Left one comment.

File: docs/public-networks/reference/cli/options.md (hunk 1 above)
Comment (usmansaleem) [reply docs/public-networks/reference/cli/options.md:698]: In production environment, a proper issues certificate is most likely to be used. This option is usually used when either self signed certificate is used or a custom CA is used to sign the server certificate.

**Pull request merged**